web/package.json

```diff
@@ -53,6 +54,10 @@
     ],
     "collectCoverageFrom": [
       "src/**/*.{js,jsx,ts,tsx}"
-    ]
+    ],
+    "moduleNameMapper": {
+      "^components(.*)$": "<rootDir>/src/components/$1",
+      "^routes(.*)$": "<rootDir>/src/routes/$1"
```

Comment:
Can we add the `theme` one as well? I think we'll be using that import fairly often.

Reply:
I learned a couple things yesterday as outlined in Issues #196 and #201. I'm converting this PR to a draft until we can discuss the changes.